### DISCUSSION TITLE
Payloads and unshareable stages?

### DISCUSSION THREAD

**Discussion by pierre-ed** [Autodesk/maya-usd#2333] [Q&A]
Hi everyone!

I am having a hard time controlling the load policy of unshearable `mayaUsdProxyShape` stages.   

With shareable stages, setting the `.loadPayloads` attribute to false before feeding in any `.filePath` does the trick perfectly.     
![20220506__payloads_01](https://user-images.githubusercontent.com/5444972/167104538-0b061f3d-854d-4454-9d6b-e89f99acde93.gif)   

On the other hand, I cannot manage to declare a proxyshape whose stage would **not** be shared, and then load its content in a deferred way: it always loads immediately. I tried to uncheck the `.shareStage` attribute first, let the stage being recreated (with its new sublayering root layer and new session layer), and then explicitly set its loading rules to `[ (</>, NoneRule) ]` before finally specifying `.filePath`, but to no avail.     
![20220506__payloads_02](https://user-images.githubusercontent.com/5444972/167104589-91e0c1c3-c02b-497e-8fea-0462d6104bb3.gif)

Indeed, a quick look at the code indicates that [the stage is reopened while ignoring any load recommendations](https://github.com/Autodesk/maya-usd/blob/005eca8a94777b407c4e4b8b4f28dd962c80ea11/lib/mayaUsd/nodes/proxyShapeBase.cpp#L908).   

Is there a technical reason behind this specific behavior (the apparent impossibility to open a unshareable stage without its payloads)? Is there any workaround? 

As always, thanks for your feedback.    
Cheers,   
   --Pierre

#### Specs

- OS & version: CentOS Linux release 7.6.1810 (Core)
- Compiler & version: gcc (GCC) 6.3.1 20170216
- Maya version: Maya 2022.3
- Maya USD: 0.17.0
- Pixar USD: 21.11

**Comment by womanyoyoyo-adsk** [comment]
Hi @pierre-ed, I'm able to reproduce this. I've logged this internally and made an issue here: #2335
